tests/lax_test.py

```diff
@@ -2031,6 +2031,21 @@ def test_conv_general_dilated_different_input_ranks_error(self):
     with self.assertRaisesRegex(ValueError, msg):
       lax.conv_general_dilated(lhs, rhs, **kwargs)
 
+  def test_reduce_window_scalar_init_value_shape_rule(self):
+    # https://github.com/google/jax/issues/4574
+    args = { "operand": np.ones((4, 4), dtype=np.int32)
+           , "init_value": np.zeros((1,), dtype=np.int32)
+           , "computation": lax.max
+           , "window_dimensions": (2, 2)
+           , "window_strides": (2, 2)
+           , "padding": "VALID"
+           , "base_dilation": (1, 1)
+           , "window_dilation": (1, 1)
+           }
+
+    msg = ("reduce_window expected init_value to have shape")
```

Comment:
Can you put the full error message here? I realize we don't always do it, but I think it's a good practice for verifying the error message is actionable and well-formatted.

Reply:
Done! Thanks!